package.json

```
@@ -20,6 +20,9 @@
     "build:core:css": "sass packages/core/style:packages/core/dist --no-source-map -s compressed",
     "build:core:esm": "tsc -p packages/core/tsconfig.json",
     "build:react:esm": "tsc -p packages/react/tsconfig.json",
+    "preprototype": "shx rm -f packages/*/prototype.js*",
+    "prototype": "concurrently yarn:prototype:*",
+    "prototype:react": "esbuild packages/react/src/index.ts --outfile=packages/react/prototype.js --bundle --define:process.env.NODE_ENV=\\\"production\\\" --target=es2015 --sourcemap --global-name=Castor",
```

💅🏻  I believe `NODE_ENV` can be defined as an environment variable and it will be loaded with dotenv, but I might be wrong

```suggestion
    "prototype:react": "NODE_ENV=production; esbuild packages/react/src/index.ts --outfile=packages/react/prototype.js --bundle --target=es2015 --sourcemap --global-name=Castor",
```

---

I actually followed the official esbuild instructions: https://esbuild.github.io/getting-started/#bundling-for-the-browser I think better to leave as they suggest, easier later to debug.